### PR TITLE
Fix/stream2 jitsi track

### PR DESCRIPTION
**Pull request by hasevr**
Note: this changes services/conference/libs/lib-jitsi-meet. You need to update submodule and yarn install at lib-jitsi-meet,
To call addTrack(), I need JitsiTrack instread of MediaTrack
Also, JitsiLocalTrack and JitsiTrack is different. Therefore, Local and Remote Participant also must be different classes.

### CODE REVIEW

File: services/conference/src/scripts/components/avatar/ConnectedAvatar.tsx [hunk 1]
```diff
@@ -14,21 +16,28 @@ const ConnectedAvatar: React.FC<ConnectedAvatarProps> = (props) => {
 
   const {
     information,
-    stream,
+    track,
     showVideo,
   } = useObserver(() => {
+    let track = undefined
+    if (participant instanceof LocalParticipant) {
```
Comment (zlguo1996) [inline services/conference/src/scripts/components/avatar/ConnectedAvatar.tsx:23]: This if else is not needed after the change on model

File: services/conference/src/scripts/models/Participant.ts [hunk 2]
```diff
@@ -20,10 +33,15 @@ export interface Information {
   avatarSrc?: string
 }
 
-export interface Stream {
-  audioStream: MediaStream | undefined
-  avatarStream: MediaStream | undefined
-  screenStream: MediaStream | undefined
+export interface Tracks {
```
[the remark this answers is not in the document]
Comment (zlguo1996) [reply services/conference/src/scripts/models/Participant.ts:36]: Dependency on JitsiTrack in StreamAvatar and StereoManager is quite dirty. Please keep audioStream: MediaStream here.
```
interface TracksBase<T extends JitsiTrack> {
 audioStream: MediaStream | undefined
 audio: T | undefined
 ...
}
```
And audioStream above could be made @computed in the store.

**Comment by zlguo1996** [comment]
Thank you @hasevr .
As I've mentioned in discord, dependency on JitsiTrack in StreamAvatar and StereoManager is quite dirty. Please see the review for detail.

**Comment by hasevr** [comment]
@zlguo1996 Thank you for review. I've fixed according to your suggestions. I believe now all fixed.

**Comment by hasevr** [comment]
Thank you. I've overlooked findRemote().

**Pull request merged**